client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/TablesApiClientFactory.java

```diff
@@ -50,6 +50,7 @@ public ApiClient createApiClient(@NonNull String baseUrl, String token, String t
       throws MalformedURLException, SSLException {
     WebClient webClient = createWebClient(baseUrl, token, truststoreLocation);
     ApiClient apiClient = new ApiClient(webClient);
+    apiClient.addDefaultHeader("Authorization", String.format("Bearer %s", token));
```

Comment:
We need to make this change in all the places where ApiClient is instantiated. For example `JobsApiClientFactory` and `HouseTablesApiClientFactory`. As HTS is internal so token might not be needed. This is one of the reason token was added as part of WebClient. Were you able to see if the the other option of overriding token on WebClient header using ApiClient also works?

Reply:
But yeah this approach should be fine as we need to add token through ApiClient and WebClient only allows through mutation which means new instance creation.

Reply:
Yeah, ApiClient also add these token to webClient headers only while preparing the request, so no need to update in webClient. 
Just that every consumer of webClient will have to do it explicitly.
Updated to add `JobsApiClientFactory` and `HouseTablesApiClientFactory` as well.

Reply:
Looks like PR is not updated with the latest changes.

Reply:
All updated now